lib/data/external/AwsClient.js

```diff
@@ -355,6 +357,35 @@ class AwsClient {
             return callback();
         });
     }
+    copyObject(request, locationType, objectGetInfo, log,
+    callback) {
+        const sourceObjectKey =
+        typeof(objectGetInfo) === 'string' ? objectGetInfo :
```

Comment:
what the situations in which is is just a string and when it is an object?

Reply:
I assume that in previous version objectGetInfo was a string representing the object's key. So we added this check for backwards compatibility. Ps: I added a comment :)